web/themes/custom/cloudy/templates/content/council-document/node--council-document.html.twig

```diff
@@ -1,12 +1,17 @@
 <div class="row">
   <div class="cloudy-content col-lg-8 col-md-12">
-    <article{{ attributes }}>
-      <div{{ content_attributes.addClass('node__content', 'clearfix') }}>
-        <h2 class="h4">{{ content.field_full_document_title.0 }}</h2>
-        <div class="council-document--type">
-          {{ content.field_is_emergency_ordinance.0 }}
-          {{ content.field_council_document_type.0 }}
-        </div>
+    <article {{ attributes }}>
+      <div {{ content_attributes.addClass('node__content', 'clearfix') }}>
+        {% if content.field_is_emergency_ordinance[0]['#markup'] == 'true' %}
+          {{ content.field_is_emergency_ordinance }}
+        {% else %}
+          {{ content.field_council_document_type }}
+        {% endif %}
+				{% if content.field_full_document_title.0 %}
```

Comment:
@brittanysmart That is some weird code formatting 😄 Wonder why your code sniffer did that?